packages/cli/src/hooks/init.ts

```diff
@@ -0,0 +1,6 @@
+module.exports = async function () {
+  // load ts-node to support dynamically loading and parsing
+  // typescript files
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  require('ts-node').register({});
```

Comment:
This might be heavy to run on every command but it doesn't seem so bad right now. In the future this might be something to abstract on to specific commands.